babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: [['@babel/preset-env', {targets: {node: 'current'}}]],
+};
```

Comment:
This file is only necessary if we remove rollup-jest. We should also uninstall rollup-jest if we do that.